packages/fluentui/react-northstar/src/themes/teams/components/Skeleton/skeletonShapeStyles.ts

```diff
@@ -0,0 +1,22 @@
+import { ICSSInJSStyle, ComponentSlotStylesPrepared } from '@fluentui/styles';
+import { SkeletonShapeStylesProps } from 'src/components/Skeleton/SkeletonShape';
+import { SkeletonVariables } from './skeletonVariables';
+
+export const skeletonShapeStyles: ComponentSlotStylesPrepared<SkeletonShapeStylesProps, SkeletonVariables> = {
+  root: ({
+    props: p,
+    variables: v,
+    theme: {
+      siteVariables: { colorScheme },
+    },
```

Comment:
Let's do with mapping in `skeletonVariables`

Reply:
done